frontend/src/components/LoginModal.tsx

```diff
@@ -46,10 +46,8 @@ export const LoginModal = ({
       renderToPortal={!isSSR}
     >
       <ModalHeading id={`${modalId}-heading`}>{titleText}</ModalHeading>
-      <div className="usa-prose">
-        <p>{helpText}</p>
-        <p className="font-sans-2xs margin-y-4">{descriptionText}</p>
-      </div>
+      <p>{helpText}</p>
+      <p className="font-sans-2xs margin-y-4">{descriptionText}</p>
```

Comment:
- Remove `.usa-prose` class since type elements now get global styles
- We still need the utility classes on the small `p` tho